deploy/linux-systemd/tigerbeetle.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=TigerBeetle server
+Documentation=https://docs.tigerbeetle.com/
+After=network-online.target
+Wants=network-online.target systemd-networkd-wait-online.service
+
+[Service]
```

Comment:
The hardening here is great - are there any other parameters we might want to set that you know of off hand? The more locked down the better :smile:

Reply:
Browsing through the systemd docs I can see a few others that might be helpful, but I haven't had the time to try them yet. I think the current hardening configurations are a pretty good start, and as I come across more things, I plan on making a PR to keep this repo in sync with the NixOS service config.